routers/web/repo/compare.go

```diff
@@ -629,9 +629,8 @@ func PrepareCompareDiff(
 	}
 
 	baseGitRepo := ctx.Repo.GitRepo
-	baseCommitID := ci.CompareInfo.BaseCommitID
 
-	baseCommit, err := baseGitRepo.GetCommit(baseCommitID)
+	beforeCommit, err := baseGitRepo.GetCommit(beforeCommitID)
```

Comment:
My understanding: the only key change is here, use `beforeCommitID` instead of `baseCommitID`. `beforeCommitID` was already prepared above. Other changes are just renaming. 😁

Reply:
👍 the best kind of bugfix